README.md

```diff
@@ -203,6 +203,24 @@ You can configure the environment variables in the workflow file like this:
           UPDATE_METHOD: "rebase"
 ```
 
+## Supported Events
+
+Automerge can be triggered by these webhook events:
+
+* `check_run`
+* `check_suite`
+* `issue_comment`
+* `pull_request_review`
+* `pull_request_target`
+* `pull_request`
+* `push`
+* `repository_dispatch`
+* `schedule`
+* `status`
+* `workflow_dispatch`
+
+For more information on when these occur, see the Github documentation on [Webhook events and payloads](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads).
```

Comment:
As a user, I didn't really know when each of these referenced events occurred.
As a contributor, I wanted to see the corresponding payloads.

Hope this helps people down the line.